TestBenches/VMStubMERouter_test.cpp

```diff
@@ -0,0 +1,111 @@
+// Test bench for VMStubMERouter
+
+// To run a different phi region, change the following:
+//          - change the top function and header file (TOP_FUNC_, HEADER_FILE_)
+//          - add the phi region in emData/download.sh, make sure to also run clean
+//          - run emData/generate_VMSMER.py to generate new top and parameters files
+
+// No macros can be defined from the command line in the case of C/RTL
+// cosimulation, so we define defaults here.
+#if !defined TOP_FUNC_
+  #define TOP_FUNC_ VMStubMERouterTop_L2PHIA
+  #define HEADER_FILE_ "VMStubMERouterTop_L2PHIA.h"
+#endif
+
+#include HEADER_FILE_
+
+#include <algorithm>
+#include <iterator>
+
+#include "FileReadUtility.h"
+
+using namespace std;
+
+const int nEvents = 5;  //number of events to run
```

Comment:
@mlarson02 Since the test vectors contain 100 events, please increase this to 100 so that we test as many events as possible by default:
```suggestion
const int nEvents = 100;  //number of events to run
```